src/main/java/io/bloco/faker/helpers/RandomHelper.java

```diff
@@ -56,6 +68,14 @@ public double range(double min, double max) {
         return random.nextDouble() * (max - min) + min;
     }
 
+    public double randLong() {
```

Comment:
Sorry, thought I will use it in Numbers but ended up not using it i've just removed it